example/src/screens/timelineCalendar.js

```diff
@@ -0,0 +1,312 @@
+// const today = new Date().toISOString().split('T')[0];
+// const fastDate = getPastDate(3);
+// const futureDates = getFutureDates(9);
+// const dates = [fastDate, today].concat(futureDates);
+
+// function getFutureDates(days) {
+//   const array = [];
+//   for (let index = 1; index <= days; index++) {
+//     const date = new Date(Date.now() + 864e5 * index); // 864e5 == 86400000 == 24*60*60*1000
+//     const dateString = date.toISOString().split('T')[0];
+//     array.push(dateString);
+//   }
+//   return array;
+// }
+
+// function getPastDate(days) {
+//   return new Date(Date.now() - 864e5 * days).toISOString().split('T')[0];
+// }
+
+const EVENTS = [
+  {
+    start: '2017-09-06 22:30:00',
+    end: '2017-09-06 23:30:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-07 00:30:00',
+    end: '2017-09-07 01:30:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-07 01:30:00',
+    end: '2017-09-07 02:20:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-07 04:10:00',
+    end: '2017-09-07 04:40:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-07 01:05:00',
+    end: '2017-09-07 01:45:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-07 14:30:00',
+    end: '2017-09-07 16:30:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-08 01:20:00',
+    end: '2017-09-08 02:20:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-08 04:10:00',
+    end: '2017-09-08 04:40:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-08 00:45:00',
+    end: '2017-09-08 01:45:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-08 11:30:00',
+    end: '2017-09-08 12:30:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-09 01:30:00',
+    end: '2017-09-09 02:00:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-09 03:10:00',
+    end: '2017-09-09 03:40:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-09 00:10:00',
+    end: '2017-09-09 01:45:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-10 12:10:00',
+    end: '2017-09-10 13:45:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+];
+
+export default class ExpandableCalendarScreen extends Component {
+  onDateChanged = date => {
```

Comment:
Don't you want to refresh the timeline events when the date changes? You should add it to the example

Reply:
@Inbal-Tish I didn't used it in example for a reason. If tomorrow someone uses this example, Expandable Calendar will be on current data and these events will be on their respective dates. In a result, events won't be visible to user.